scripts/runfit.jl

```diff
@@ -0,0 +1,92 @@
+working_dir = "/home/vekondra/ACEpotentials.jl/scripts"
+
+using Pkg
+Pkg.activate(joinpath(working_dir, "."))
```

Comment:
I also don't like this activation here. In general a user should create a working directory for *their* project, where `ACEpotentials` is added (to the local `Project.toml`). This will then be the activated project.